packages/forms/src/elements/number/number.mdx

```diff
@@ -112,10 +123,13 @@ Accepted config props: FlexProps, SpaceProps
 
 ### Props
 
-| Property | Required | Type    | Description                                  |
-| -------- | -------- | ------- | -------------------------------------------- |
-| cfg      | false    | object  | FlexProps & SpaceProps                       |
-| disabled | false    | boolean | Disable checkbox                             |
-| testId   | false    | string  | data attribute for testers                   |
-| label    | true     | string  | Checkbox description                         |
-| hint     | false    | string  | More detailed description about the checkbox |
+| Property | Required | Type     | Description                                          |
+| -------- | -------- | -------- | ---------------------------------------------------- |
+| cfg      | false    | object   | FlexProps & SpaceProps                               |
+| disabled | false    | boolean  | Disable checkbox                                     |
+| testId   | false    | string   | data attribute for testers                           |
+| label    | true     | string   | Checkbox description                                 |
+| hint     | false    | string   | More detailed description about the checkbox         |
+| callback | false    | function | callback function to be executed after onChange      |
+| after    | false    | string   | emulates text passed to ::after pseudo-selector      |
+| decimals | false    | number   | the number of decimals used for formatting the value |
```

Comment:
It would be clearer to say "decimal places" here rather than "decimals", as that's how we refer to them in English. You could change it in the code to `decimalPlaces` too. The code all looks fine though.